android-client/app/src/test/java/com/example/habits/MainDIComponentTest.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.habits
+
+import org.koin.core.context.loadKoinModules
+import org.koin.core.context.startKoin
+
+const val BASE_URL = "localhost:3000/api/v1/"
```

Comment:
should this be an environment variable or some sort of config var?

Reply:
configuration variable